src/main/java/Duke.java

```diff
@@ -24,17 +28,36 @@ public static void main(String[] args) {
                 items.get(order - 1).setMark(false);
                 System.out.println("____________________________________________________________\n" + "  " + " OK, I've marked this task as not done yet:\n" + "[X] " + items.get(order - 1).getDescription() + "\n" + "____________________________________________________________\n");
```

Comment:
better to use strings that you are constantly using (e.g. underline) into a new variable